roxctl/central/generate/generate.go

```diff
@@ -203,6 +203,11 @@ func createBundle(logger logger.Logger, config renderer.Config) (*zip.Wrapper, e
 func OutputZip(logger logger.Logger, config renderer.Config) error {
 	logger.InfofLn("Generating deployment bundle...")
 
+	if config.EnablePodSecurityPolicies {
```

Comment:
The logic of the if-case is still duplicated.
Moving this if-case to a function `printPSPInfoLog(config.EnablePodSecurityPolicies)` and call it from all other occurrences of this clause.